<!--
Copyright (C) 2013   Bob Rutledge

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.
You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>
and open the template in the editor.
-->
<html>
    <head>
        <title>Generating an OCF</title>
        <link rel="stylesheet" href="nbdocs:/org/netbeans/modules/usersguide/ide.css" type="text/css">
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    </head>
    <body>
        <h1 align="center">Generating an OCF</h1>
		<p>Note that it is necessary to manually enter an OCF into the 
		experiment database window before the program is able to calculate the 
		number of target molecules. </p>
		<p align="center">
		<img border="0" src="images/ocf2.gif" width="424" height="215"></p>
		<p>It is also important to note that an OCF is 
		specific to how the amplification reactions are setup, that is, specific 
		to the optics of an individual assay. Thus changing, for example, the 
		enzyme formulation or the type of reaction vessel requires that a new 
		OCF be determined, which also generally required a new experiment 
		database to be created for analysis of data generated by this new setup. </p>
		<p><b>Quick start using a crude OCF estimate: 5% of the run&#39;s average Fmax<br>
		</b>Although this method is not generally recommended, empirical observation has 
		indicated that a lambda gDNA-derived OCF (see below) is often close to 5% of average F<sub>max</sub> 
		generated by the profiles within a run. The 
		demonstration experiment database provides a nice illustration, 
		in that 5% the average F<sub>max</sub> (~2.5x10<sup>6</sup>) is 
		about 125,000, which is close to the average OCF generated by the 
		corresponding demonstration calibration database (~120,000). </p>
		<p><b>Alternative quantitative standards<br>
		</b>Although the ubiquitous nature of lambda gDNA makes it an attractive 
		choice for optical calibration, any DNA target of known quantity can be 
		used to derive an OCF. For example, profiles from a standard curve can 
		be used to determine an OCF, simply by importing the profiles into a new 
		experiment database and&nbsp; repeatedly entering OCF values until 
		correlation to the predicted target quantities are maximized. While 
		somewhat clumsy, this approach can provide insights into how the 
		application of an OCF impacts target quantification.</p>
		<p>That is, while these approaches may introduce some error in absolute scale, this 
		error is both linear and equally applied across all quantitative 
		determinations. Thus, a 20% error in OCF generates an equivalent 20% 
		error in all target quantifications to which this OCF is applied. As 
		such, relative differences are not impacted by errors in OCF, and in 
		general, such errors are small in relation to the interpretation of qPCR 
		data. In gene expression profiling, for example, differences in 
		transcript quantity of less than 1-fold are generally considered to have 
		modest, if any, biological significance. Although it is certainly 
		possible for an aberrant OCF to introduce such a large error, in 
		practice such errors are uncommon.&nbsp;&nbsp; </p>
		<p><b>Using lambda gDNA for generating an OCF<br>
		</b>In combination with the calibration database window, an&nbsp; OCF is 
		generated by amplifying a known quantity of lambda gDNA using the lambda 
		primer-pair CAL1. Importantly, the amplification setup must be identical 
		to that used for sample amplification, which can be accomplished by 
		simply treating these reactions as an additional sample within a run. 
		Including calibration reactions into every run also provides a 
		convenient method for quality control, in that any aberrations in 
		reaction setup or instrumentation can be identified by aberrations in 
		both the kinetics of the resulting calibration profiles and in the OCF 
		values they produce. </p>
		<p>The resulting 
		amplification profiles are referred as &quot;calibration profiles&quot;, which are 
		processed within the calibration database window. </p>
		<p>CAL1 5&#39; primer: AGACGAATGCCAGGTCATCTGAAACAG<br>
		CAL1 3&#39; primer: CTTTTGCTCTGCGATGCTGATACCG</p>
		<p>Lambda gDNA is diluted with 10 mM Tris using siliconized microfuge 
		tubes, to a final concentration of 100 femtograms per
		<font face="Times New Roman">µ</font>l, which is equivalent to 1,876 
		genomes. Note that bulk preparations of&nbsp;1000 femtogram of lambda gDNA that also 
		includes a X10 concentration of primers (e.g. 5000 nM/<font face="Times New Roman">µ</font>l 
		for a 500 nM final concentration in a 10 <font face="Times New Roman">µ</font>l reactions) 
		can be used to standardize optical calibration over long 
		time periods , in addition to providing a single source of calibration 
		mix that can be applied to multiple instruments or reaction setups (e.g. 
		different enzyme formulations). </p>
		<p>A standard reaction setup that has been found to be effective, uses as 
		a reaction volume of 10 <font face="Times New Roman">µ</font>l 
		containing 100 fg of lambda gDNA, amplified in a 
		white plate sealed with film. Typically, four replicate CAL1 reactions are included 
		in 
		each run.</p>
		<p>As described in the
		<a href="../explorer_panel/calibration_database_window.html">optical 
		calibration explorer window</a> overview, calibration profiles are stored 
		in a dedicated database, from which an average OCF is derived from multiple runs. As described in the
		<a href="../explorer_panel/experiment_database_window.html">experiment 
		explorer window overview</a>, an average OCF is manually entered, which 
		is then applied to all runs within the experiment database.</p>
		<p><b>Mathematics</b></p>
		<p>Optical calibration quantifies the fluorescence intensity of an assay 
		based on the average F<sub>0</sub> generated by a calibration profile 
		using the equation:</p>
		<p align="center">
		<img border="0" src="images/lambda_ocf_equation.gif" width="168" height="57"></p>
		<p>
		M<sub>0</sub> is 
		equivalent to the mass of the amplicon region within the target 
		expressed in nanograms of dsDNA, which for lambda is calculated using the 
		equation:</p>
		<p align="center">
		<img border="0" src="images/lambda_mo.gif" width="219" height="62"></p>
		<p>
		where &quot;ng Lambda&quot; is the nanograms of lambda gDNA that was amplified, A<sub>S
		</sub>is the amplicon size and 48,502 bp is the genome size of lambda in base 
		pairs. </p>
		<p align="left">
		As described in the <a href="optical_calibration_overview.html">optical 
		calibration overview</a>, an OCF is used to convert target quantities 
		expressed in fluorescence units to the number of target molecules. </p>
    </body>
</html>
